style.css

```diff
@@ -4,3 +4,72 @@ body {
   text-align: center;
   font-family: 'Lato', sans-serif;
 }
+
+.nav-bar {
+  display: flex;
+  border: 3px black solid;
+  width: 600px;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: space-around;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
```

Comment:
- [ ] Kindly consider removing the exact `width` from this element so that navbar would have full width at all screen sizes so that it would be similar to the wireframe given.
your design
![qwe](https://user-images.githubusercontent.com/96683843/220720709-bff71355-bc33-46bb-8ace-4e68a7dae528.jpg)
given wireframe
![fd](https://user-images.githubusercontent.com/96683843/220720617-4bdb690b-7c0f-4018-bbe6-8f922162768b.jpg)